servers/sunrpc/handshake_prog.go

```diff
@@ -6,7 +6,8 @@ import (
 	"path"
 
 	"github.com/gluster/glusterd2/utils"
-
+	"github.com/gluster/glusterd2/servers/sunrpc/program"
+	"github.com/gluster/glusterd2/servers/sunrpc/serialize"
```

Comment:
I wanted to do this, move serialise out. As mentioned above, serialise can be moved to a package in source's root dir - something like package `dict`.